translate/src/main/java/com/google/cloud/translate/samples/TranslateText.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.translate.samples;
+
+import com.google.cloud.translate.Detection;
+import com.google.cloud.translate.Language;
+import com.google.cloud.translate.Translate;
+import com.google.cloud.translate.Translate.TranslateOption;
+import com.google.cloud.translate.Translation;
+import com.google.cloud.translate.testing.RemoteTranslateHelper;
+import com.google.common.collect.ImmutableList;
+
+import java.io.PrintStream;
+import java.util.List;
+
+public class TranslateText {
+  private static final Translate TRANSLATE = RemoteTranslateHelper.create().options().service();
```

Comment:
Probably want to pass the `Translate` object to each individual function here rather than creating it statically, also. That way it is more clear that developers can create the object as part of a normal lifecycle.

Consider even having each function create a Translate object, depending on how these snippets will be used, that could be the most clear.